CHANGELOG.md

```diff
@@ -5,11 +5,19 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 This changelog adheres to [Keep a CHANGELOG](http://keepachangelog.com/).
 
 ## [Unreleased]
+## [0.4.2] - 2015-12-22
+### Fixed
+- Fixed problem when tools use out of tree builds
+- Vanagon no longer changes group ownership on
+  Solaris 10 installs
+
+### Changed
+- Added lock_manager gem to vanagon project
```

Comment:
Added Hardware engine is probably the more important part of that statement to vanagon users.